x/emissions/keeper/inference_synthesis/network_regrets.go

```diff
@@ -138,7 +144,19 @@ func GetCalcSetNetworkRegrets(
 		if err != nil {
 			return errorsmod.Wrapf(err, "Error setting inferer regret")
 		}
-		if !newParticipant {
+
+		shouldAddWorkerRegret, err := isExperiencedWorker(
```

Comment:
do we need to make research aware of the change here for the purposes of keeping the paper aligned with the codebase? 

@jmdkastro FYI now we will only use "experienced" workers for Network Regrets

Reply:
Yes, I think the research team has already tested this solution